webui/react/src/components/TableBatch.tsx

```diff
@@ -25,7 +27,12 @@ const TableBatch: React.FC<Props> = (props: PropsWithChildren<Props>) => {
     <div className={classes.join(' ')}>
       <div className={css.container}>
         <div className={css.actions}>{props.children}</div>
-        <div className={css.message}>{message}</div>
+        <div className={css.actions}>
+          <div className={css.message}>{message}</div>
+          {props.onClear &&
+            <Button onClick={props.onClear}>Clear Selected</Button>
+          }
+        </div>
```

Comment:
**suggestion:**
```
<div className={css.message}>{message}</div>
{props.onClear && (
  <div className={css.clear}>
    <Button onClick={props.onClear}>Clear Selected</Button>
  </div>
)}
```
will require `css.clear` to be defined and probably a little tweak of the other css. Right now there is a big space to the right to the clear button.